GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -124,7 +125,23 @@ private void UnhandledGVFSExceptionHandler(ITracer tracer, object sender, Unhand
 
         private JsonTracer CreateTracer(GVFSEnlistment enlistment, EventLevel verbosity, Keywords keywords)
         {
-            JsonTracer tracer = new JsonTracer(GVFSConstants.GVFSEtwProviderName, "GVFSMount");
+            string enlistmentId = null;
+            string mountId = null;
+
+            GitProcess git = new GitProcess(enlistment);
+            GitProcess.Result configResult = git.GetFromLocalConfig(GVFSConstants.GitConfig.EnlistmentId);
+            if (!configResult.HasErrors)

```

Comment:
@jrbriggs do we want these to be silent errors?

Reply:
@kewillford under what cases would these be unset? Corrupted gitconfig, and user tampering? We wouldn't recover here, I assume, but in the verbs?

Reply:
If we are unable to get the config values then the enlistmentId and/or mountId will be left null and will be blank in the Json at least from when I was testing.

Reply:
So back to my question :-)... We've answered that the code will continue to work if these values are missing, but do we _want_ this to be a silent error, or should we blow up and root cause the issue? Worst case here is that logging/telemetry data goes missing because we ignored some error that we should have been able to handle.

Reply:
That depends on what recovery looks like. Mount id will be reset on remount. @kewillford what about enlistment id?

Reply:
The EnlistmentId will get set again on mount using the value in the RepoMetadata and the data that will be missing is the MountId and/or EnlistmentId which would make querying and joining data more difficult but we will not be missing events.

Reply:
@jrbriggs to me that is a strong argument to blow up here. If the `gvfs mount` verb was supposed to generate the mount id, and here in the `gvfs.mount` process we can't find it, something has gone very wrong. I'd hate to hide more serious bugs by ignoring these errors.

I don't think this is a release blocker, but I would strongly favor a hard failure here.

Reply:
@sanoursa I chatted with @kewillford offline, and the summary of that conversation is captured in #398, which we'll get into the next release.